<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingDialog</class>
 <widget class="QDialog" name="settingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QSpinBox" name="ratioSpinBox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QSlider" name="thresholdSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QRadioButton" name="jpgButton">
       <property name="text">
        <string>JPG</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="thresholdSpinBox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ratio:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="pngButton">
       <property name="text">
        <string>PNG</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QSlider" name="ratioSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ratioSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ratioSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>thresholdSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ratioSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ratioSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>thresholdSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>settingDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>settingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
